docker/workers.conf

```diff
@@ -0,0 +1,11 @@
+[program:laravel-worker]
+process_name=%(program_name)s_%(process_num)02d
+command=php /var/www/html/artisan queue:work --queue=video-transcoding
+autostart=true
+autorestart=true
+stopasgroup=true
+killasgroup=true
```

Comment:
when does this come into account

Reply:
If true, when resorting to send SIGKILL to the program to terminate it send it to its whole process group instead, taking care of its children as well.

Might make sense to set this to false?